lib/web_console/view.rb

```diff
@@ -27,7 +27,7 @@ def render_inlined_string(template)
 
     # Escaped alias for "ActionView::Helpers::TranslationHelper.t".
     def t(key, options = {})
```

Comment:
Hi :-) At first, I want to replace `\n` with `\\n`, since it could cause a syntax error in JavaScript. And one more thing, I want to replace `"` with `\"`, because the method could return a HTML tag with some attributes when the key is not found (but I think it happens only in the development).